rust/src/common/options.rs

```diff
@@ -30,7 +30,7 @@ use std::time::Duration;
 /// ```rust
 /// let options = Options::new().infer(true).explain(true);
 /// ```
-#[derive(Clone, Debug, Default)]
+#[derive(Clone, Copy, Debug, Default)]
 pub struct Options {
```

Comment:
`Options` is 88 bytes of data which doesn't manage any memory. Marking it as trivially copyable simplifies its usage.